libs/platform/src/lib/components/split-menu-button/split-menu-button.component.html

```diff
@@ -16,9 +18,11 @@
         fd-button
         [attr.id]="secondaryId"
         [attr.name]="name"
```

Comment:
We're not supposed to add 'id' attribute for element in library , Please check with Kevin on this.

Reply:
Adding an `id` here is fine. It will help QA test distinguish between the main button and the menu button.